test/functional/mempool_packages.py

```diff
@@ -69,11 +69,22 @@ def run_test(self):
         fee = Decimal("0.0001")
         # MAX_ANCESTORS transactions off a confirmed tx should be fine
         chain = []
+        wallet_unspent = self.nodes[0].listunspent(minconf=0)
```

Comment:
add a quick test making sure these fields don't exist when transaction confirmed/not in mempool?

Reply:
> add a quick test making sure these fields don't exist when transaction confirmed/not in mempool?

Good idea (worst case can be done as a follow-up, along with adding logging to the tests etc.)